extensions/resource-deployment/src/interfaces.ts

```diff
@@ -163,17 +163,17 @@ export interface FieldInfo {
 	fontStyle?: FontStyle;
 }
 
-export enum LabelPosition {
+export const enum LabelPosition {
 	Top = 'top',
 	Left = 'left'
 }
 
-export enum FontStyle {
+export const enum FontStyle {
 	Normal = 'normal',
 	Italic = 'italic'
 }
 
-export enum FieldType {
+export const enum FieldType {
 	Text = 'text',
```

Comment:
Why did you make these all const?

Reply:
Because that inlines them which is a good idea for string based enums.

https://mariusschulz.com/blog/string-enums-in-typescript